### PR TITLE
Geocoord status doc

### PR DESCRIPTION
**Pull request by AlainRobertAtBentley**
Added required documentation for the Geographic Coordinate Reference Systems and datum conversion status.
It is a bit wordy but not unjustifyably so.

### CODE REVIEW

File: core/common/src/GeoCoordinateServices.ts [hunk 1]
```diff
@@ -9,14 +9,46 @@
...
   NoDatumConverter = 25,
+  /** Indicates that a problem occurred during vertical datum conversion. This may the result of the incapacity to access the
+   *  Geoid separation data or vertical datum differential data (such as used by NGVD29 to NAVD88 conversion).
+   *  The horizontal coordinates returned are valid but the elevation ordinate will be returned unchanged or partially changed.
+   *  This status should be somehow reported so actions can be performed to provide the missing information.
+   */
   VerticalDatumConvertError = 26,
+  /** General inner conversion engine error. Coordinates returned are invalid and should be discarded */
   CSMapError = 4096,
   Pending = -41556,
```
Comment (pmconne) [inline core/common/src/GeoCoordinateServices.ts:54]: What does "pending" mean?

Comment (AlainRobertAtBentley) [reply core/common/src/GeoCoordinateServices.ts:54]: @pmconne I have no idea what pending means and I have not located anywhere where  this code could be returned. My assumption is that Barry wanted to reserve a status for pending conversion requests but such a status should be returned for the whole batch of requests not for every coordinate. I beleive there are other conventions for pending batch requests that are general. Anyway, we can either eliminate the status or leave it temporarily undocumented.

Comment (pmconne) [reply core/common/src/GeoCoordinateServices.ts:54]: Thanks, I found no usage either. We should eliminate it.

Comment (AlainRobertAtBentley) [reply core/common/src/GeoCoordinateServices.ts:58]: The pending status was set in the frontend geo-service to mark coordinates that actually needed conversion from the backend as opposed to those solved from the cache. Review wording as I am not sure of terminology.

Comment (pmconne) [reply core/common/src/GeoCoordinateServices.ts:58]: Ah I see. Documentation looks clear to me.

**Review by pmconne — APPROVED** [review]
